src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/index.tsx

```diff
@@ -211,6 +224,21 @@ export default function NftDrawer({
           >
             <NftMarketplaceCustomizationForm form={marketplaceForm} />
           </MinimalCollapse>
+          <MinimalCollapse
+            header={
+              <>
+                <Trans>Purchase confirmation popup</Trans>
```

Comment:
**question** I wonder if theres a better word than `purchase` 🤔 

`Payment success popup` is the best i got.

Reply:
Yeah I think since this is in the NFT rewards sections it makes enough sense